WordPress/WordPressTest/BloggingPromptsServiceTests.swift

```diff
@@ -138,49 +133,28 @@ final class BloggingPromptsServiceTests: CoreDataTestCase {
     }
 
     func test_fetchPrompts_givenNoParameters_assignsDefaultValue() throws {
```

Comment:
This test only needs to assert that the default date matches the URL parameter – no need for `differenceInDays`